### PR TITLE
docs(readme): refactor, cleanup, & styling

### PR DESCRIPTION
**Pull request by LeoDog896**
## Description

This:
- Fixes some typos
- Removes the broken Codacy badge & replaces the broken heroku badge
- Formats it using [deno's code formatter](https://deno.land/manual@v1.29.3/tools/formatter)
- Adds syntax highlighting to most code blocks
- Uses opinionated markdown syntax (preferring `#` over `==========`, `-` over `*` for lists) -- this causes less markdown errors and should be preferred.

## Motivation and Context

The README, at first glance, can look very confusing. This aims to alleviate some of the pain of that.

## Types of changes

- [x] Docs change / refactoring

### CODE REVIEW

**Comment by LeoDog896** [comment]
P.S: would it be appropriate to find contact info? I can't find an IRC chat, discord, or your email whatsoever, so there's no way to report security vulnerabilities or chat and the like. If you can, please provide some info!

**Comment by mpromonet** [comment]
Hi,

Thanks for improvement of readme, I will look to that. The heroku is no more used since there is no more freeplan, I am testing azure deployement.

To discuss some feature, github is a good way, I add the discussion feature as you suggested.

Best Regards,
Michel.

File: README.md [hunk 1]
```diff
@@ -1,295 +1,397 @@
...
+  ```sh
+  docker run -p 8000:8000 -it mpromonet/webrtc-streamer -n raspicam -u rtsp://pi2.local:8554/unicast
+  ```
+- run the container giving config.json file:
 
+  ```sh
+  docker run -p 8000:8000 -v $PWD/config.json:/app/config.json mpromonet/webrtc-streamer
+  ```
 
+## License
```
Comment (mpromonet) [inline README.md:394]: LICENSE is already presented by github in the side panel.

Comment (LeoDog896) [reply README.md:394]: Oops -- yeah, that's redundant, removing now.

**Pull request merged**